gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcOrganizationRepository.java

```diff
@@ -126,6 +127,29 @@ public Set<Organization> findByHrids(Set<String> hrids) throws TechnicalExceptio
         }
     }
 
+    @Override
+    public Optional<Organization> findByCockpit(String cockpitId) throws TechnicalException {
+        LOGGER.debug("JdbcOrganizationRepository.findByCockpit({})", cockpitId);
+        try {
+            List<Organization> organizations = jdbcTemplate.query(
+                getOrm().getSelectAllSql() + " where cockpit_id = ?",
+                getOrm().getRowMapper(),
+                cockpitId
+            );
+
+            final Optional<Organization> organization = organizations.stream().findFirst();
```

Comment:
maybe I miss something, but why taking the first only, for a cockpit ID we can have multiple orga (or env) no ? 
So how to unlink everything for a cockpit id ? 

EDIT: from tests, I see the cockpitId are containing info about env or org, if it is the case in real life then no problem to find the first

Reply:
There is a cockpitId for organization and a cockpitId for environment. When the organization/environment is created through cockpit, the cockpitId and _id is the same. However for the default organization/environment, the _id is DEFAULT and when the user link an installation to the default env, the command will update the cockpitId in APIM.

So basically, the cockpitId should be unique

Reply:
Good point ! 
I agree this part is strange, but we have the same implementation In EnvRepo... 
And in fact, with the test describe in https://gravitee.atlassian.net/browse/APIM-3590 we have no unique cockpitId in collection... @jgiovaresco do you have time to discuss on this part today ?